index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <a class="logo"> My Logo</a>
+    <img src="assets/Union.jpg" alt="menu-bar" />
+    <section class="intro-wrapper">
```

Comment:
- [ ] [OPTIONAL] Kindly wrap the whole Headline section in ``` section``` tag not just the header (```h1```)